php/libraries/User.class.inc

```diff
@@ -252,6 +252,24 @@ class User extends UserPermissions
     }
 
     /**
+     * Returns all user's sites that are StudySites
+     *
+     * @return array
+     */
+    function getStudySites()
+    {
+        $site_arr         = $this->userInfo['CenterIDs'];
+        $user_study_sites = array();
+        foreach ($site_arr as $key => $val) {
+            $site[$key] = &Site::singleton($val);
```

Comment:
why wouldn't it be acceptable? I think it's fine, but is that `&` supposed to be there?